src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -180,7 +183,8 @@ public DocumentDatabase(string name, RavenConfiguration configuration, ServerSto
                 TombstoneCleaner = new TombstoneCleaner(this);
                 DocumentsStorage = CreateDocumentsStorage(addToInitLog);
                 CompareExchangeStorage = new CompareExchangeStorage(this);
-                AiIntegrations = new AiIntegrationsController(this);
+                EmbeddingsGeneratorQueries = new EmbeddingsGenerator(this,_logger, DatabaseShutdown, EmbeddingsGenerator.Mode.Query);
```

Comment:
not sure if passing this logger is a good idea, it will not have appropriate type set, it will look like the logs come from DocumentDatabase, we have DatabseLoggersContext